Be more specific about agendas for meetings (#691)
2i2c-org/team-compass: operations/meetings.md

- **Take notes and make them accessible**.
  Meetings must have written notes that are accessible to others on the team.
  These must be made available to the team reasonably soon after the meeting.

- **Meeting agendas must be available at least 36 hours (not counting weekends) ahead**.
  Agendas should:

>   1. List any specific **decisions** people want to make progress on during the meeting.

Reviewer: ```suggestion
  1. Any **questions** that people wish to discuss and answer during the meeting.
  2. Any **decisions** people want to make progress on during the meeting, and where the final decision will be made.
```

  2. Provide ideas for topics of possible discussion.

  Agendas should be concrete enough that people who can not be synchronously

>   present can make comments on it, as well as make choices on wether they

Reviewer 2: ```suggestion
  present can make comments on it, as well as make choices on whether they
```

Author: wether it is whether or wether, but maybe it's weather? hmm, perhaps wether is not actually a word :D

Reviewer 2: My friend thought a "sweater" was actually "sweather" for the longest time and it was too funny to correct

Author: sweeter sweather weather

  would like to attend or not.

>   If a meeting does not have an agenda 36 hours (not counting weekends) in
>   advance, then team members may assume it is cancelled and rescheduled.

Reviewer: ```suggestion
  **If there's not an agenda 36 hours before, assume the meeting is cancelled**.
  Meeting facilitators are responsible for collecting potential agenda items from others ahead of time, and setting the meeting agenda 36 hours prior to the meeting start.
  If there is no agenda at this point, the meeting should be deemed as cancelled.
  The meeting facilitator should inform the team and delete the calendar event.
```

- **Take notes and make them accessible**.
  Meetings must have written notes that are accessible to others on the team.
  These must be made available to the team reasonably soon after the meeting.

- **Meeting agendas must be available at least 36 hours (not counting weekends) ahead**.
  Agendas should list:
  1. Any **questions** that people wish to discuss and answer during the meeting.
  2. Any **decisions** people want to make progress on during the meeting, and where the final decision will be made.
  3. Any **topics** to discuss in the meeting, with enough detail to help others decide if they wish to participate.

  Agendas should be concrete enough that people who can not be synchronously
  present can make comments on it, as well as make choices on whether they
  would like to attend or not.
  Provide links to the **source of truth for discussion**, as well as any context needed to meaningfully participate.

>   **If there's not an agenda 36 hours before, assume the meeting is cancelled**.
>   Meeting facilitators are responsible for collecting potential agenda items from others ahead of time, and setting the meeting agenda 36 hours prior to the meeting start.
>   If there is no agenda at this point, the meeting should be deemed as cancelled.
>   The meeting facilitator should inform the team and delete the calendar event.

Reviewer: ```suggestion
- **If there's not an agenda 36 hours before, the facilitator should cancel the meeting**.
  Meeting facilitators should collect _potential_ agenda items from others ahead of time, and _set_ the agenda at least 36 hours prior to the meeting start.
  If there is no set agenda at this time, facilitator should inform the team in Slack and delete the calendar event.
```

Reviewer: how about this as an iterative improvement and we can discuss others in another thread / in the team meeting?
